Sources/MediaViewer/Transition/Animator/Animator.swift

```diff
@@ -1,10 +1,11 @@
 import UIKit
 
+@MainActor
 open class Animator: NSObject, UIViewControllerAnimatedTransitioning {
     open func transitionDuration(
         using transitionContext: UIViewControllerContextTransitioning?
     ) -> TimeInterval {
-        CATransaction.animationDuration()
+        0.46
```

Comment:
This was just carefully chosen or does it correlate with something?

Reply:
carefully chosen